vispy/app/tests/test_app.py

```diff
@@ -226,19 +226,19 @@ def fake(event):
         frag = "void main (void) {gl_FragColor = pos;}"
         program = Program(vert, frag)
         assert_raises(RuntimeError, program.glir.flush, context.shared.parser)
-        
```

Comment:
ahh, sorry. I've been making them partly to slowly move us toward no whitespace (I think we agreed on that in the sprint) and also because it really helps when coding (cleaner introspection), but I suppose it should probably wait for one big PR after most of the others are merged